packages/design-system/src/components/popup/index.js

```diff
@@ -163,6 +192,7 @@ Popup.propTypes = {
   zIndex: PropTypes.number,
   spacing: PropTypes.object,
   isOpen: PropTypes.bool,
+  noOverFlow: PropTypes.bool,
   fillWidth: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
   fillHeight: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
   onPositionUpdate: PropTypes.func,
```

Comment:
I think `topOffset` should remain optional because from what I understand `Popup` that lives in `design-system` is not aware which CMS(no CMS) it is in. If not passed then it sets to 0.